sdk/v1/client.go

```diff
@@ -138,7 +138,7 @@ func (cli *Client) PatchJob(ctx context.Context, headers client.Headers, jobID s
 		ETag: respETag,
```

Comment:
This can be modified as
respHeaders := client.RespHeaders{
  Etag: respHeader.Get(ETagHeader)
}

Reply:
good spot - done